server/package-lock 2.json

```diff
@@ -0,0 +1,2004 @@
+{
```

Comment:
this file shouldn't be here

Reply:
Hmm... Don't know how that got there. Deleted!